telemetry/definitions/commonDefinitions.json

```diff
@@ -142,6 +147,63 @@
             "name": "aws_reportPluginIssue",
             "description": "Report an issue with the plugin"
         },
+        {
+            "name": "cloudwatchlogs_openGroup",
+            "description": "Open the CloudWatch Logs group window",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudwatchlogs_openStream",
+            "description": "Open a CloudWatch Logs stream in the window",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_downloadToFile",
+            "unit": "Bytes",
+            "description": "Download a log stream to a file. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openStreamInEditor",
+            "unit": "Bytes",
+            "description": "Download a log stream to open in editor. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openCurrentInEditor",
+            "unit": "Bytes",
```

Comment:
Why is unit bytes if it refers to number of lines?

Reply:
Because I forgot to change it :P . I don't think number of lines will help us capture anything so just removed